openeo-geotrellis/src/main/scala/org/openeo/geotrellis/zarr/ZarrWriter.scala

```diff
@@ -176,5 +199,16 @@ object ZarrWriter {
     val split = path.split("/").last
     split.substring(0,split.length-5)
   }
+
+  private def checkBandNames(bandNames: Array[String]):Unit = {
+    for (bandName <- bandNames){
+      if (bandName == "Undefined") throw new IllegalArgumentException("Band names are not defined")
```

Comment:
Add context to message, process name and format

Reply:
Can you also add the process name (save_result) and output format (Zarr) to all messages?